lib/css/modules/_orangered.scss

```diff
@@ -25,6 +25,13 @@
 	}
 }
 
+.res-orangered-hideEmptyModmailButton {
```

Comment:
Can you add an option to orangered.js to activate this rule? You can model it on the [`hideModmail` option](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/e70d7963907dd8e2ec18db079fecf398de13cd6b/lib/modules/orangered.js#L75).